ophyd/epics_motor.py

```diff
@@ -194,7 +194,7 @@ def home(self, direction, wait=True, **kwargs):
         direction = HomeEnum(direction)
 
         self._started_moving = False
-        position = (self.low_limit+self.high_limit)/2
+        position = (self.low_limit + self.high_limit) / 2
```

Comment:
I agree with this change but the fact this is the default behavior is odd

Reply:
Open an issue, please